src/main/java/org/wildfly/security/_private/ElytronMessages.java

```diff
@@ -72,9 +70,6 @@
     @Message(id = 1, value = "WildFly Elytron version %s")
     void logVersion(String versionString);
 
-    @Message(id = 2, value = "Parse error")
```

Comment:
It's okay (in my opinion) to leave gaps, however, there should be a comment in their place so that they're visibly obvious.